sw/device/lib/testing/i2c_testutils.c

```diff
@@ -164,3 +166,56 @@ bool i2c_testutils_target_check_wr(dif_i2c_t *i2c, uint8_t byte_count,
 
   return i2c_testutils_target_check_end(i2c, cont_byte);
 }
+
+void i2c_testutils_pinmux(const dif_pinmux_t *pinmux, uint8_t kI2cIdx) {
```

Comment:
Please rename this so it's consistent with the naming scheme and has a verb.
Maybe: "i2c_testutils_connect_i2c_to_pinmux_pins"